pkg/cloudprovider/vsphere/cloud.go

```diff
@@ -20,6 +20,8 @@ import (
 	"io"
 	"runtime"
 
+	"k8s.io/cloud-provider-vsphere/pkg/cloudprovider/vsphere/loadbalancer"
```

Comment:
Group this import with other imports starting with `k8s.io/cloud-provider-vsphere`